README.md

```diff
@@ -62,29 +60,13 @@ import { ThemeProvider } from '@itwin/itwinui-react';
 
 const App = () => (
   <>
-    <ThemeProvider theme='dark' />
-    // Your code goes here.
+    <ThemeProvider theme='dark'>
+      {/* Your components go here. */}
+    </ThemeProvider>
   </>
 );
 ```
 
-- The `useTheme` hook also provides the same functionality as `ThemeProvider`.
```

Comment:
I've removed `useTheme` from the README because our users should wrap their app in `ThemeProvider` whenever possible.